website/docs/markdown-features.mdx

```diff
@@ -101,22 +101,23 @@ Documents use the following markdown header fields that are enclosed by a line `
 - `hide_title`: Whether to hide the title at the top of the doc. By default it is `false`.
 - `sidebar_label`: The text shown in the document sidebar and in the next/previous button for this document. If this field is not present, the document's `sidebar_label` will default to its `title`.
 - `custom_edit_url`: The URL for editing this document. If this field is not present, the document's edit URL will fall back to `editUrl` from options fields passed to `docusaurus-plugin-content-docs`.
-- `description`: Description of this document page.
+- `description`: Description meta tag for the document page, for search engines.
```

Comment:
```suggestion
- `description`: Description meta tag for the document page, for search engines.. If this field is not present, it will default to the first line of the contents.
```

Reply:
check ✅